index.md

```diff
@@ -9,20 +9,22 @@ layout: default
 A national, distributed computing partnership for data-intensive research
 
 <div id="osg-special-banner">
-  <p class="special-banner-1"><strong>Extra OSG Support for COVID-19 Research</strong></p>
+  <p class="special-banner-1"><strong>We are here to help with your <a href="https://www.nsf.gov/pubs/2021/nsf21528/nsf21528.htm/">Campus Cyberinfrastructure (CC*) Proposal!</a> Please contact us at <a href="mailto:cc-star-proposals@opensciencegrid.org">cc-star-proposals@opensciencegrid.org</a></strong></p>
   <p class="special-banner-1" style="margin-bottom: 0.5ex;">
```

Comment:
Generally, I set up the styles so that only the first paragraph had the special-banner-1 class. The rest used special-banner-2 and sometimes added extra spacing. It would be good to look at the banner from last year carefully.